frontend/packages/kubevirt-plugin/src/components/cdi-upload-provider/upload-pvc-popover.tsx

```diff
@@ -64,7 +64,7 @@ export const UploadPVCPopover: React.FC<PVCUploadStatusProps> = ({ pvc }) => {
       case UPLOAD_STATUS.UPLOADING:
         return {
           title: `Uploading`,
-          body: 'Please do not close this window, you can keep navigating the app freely.',
+          body: "Please don't close this browser tab until the upload is complete.",
```

Comment:
same thing as above - I like educating the user on their ability to navigate around. "App" probably isn't the best word though. Maybe 
`Please don't close this browser tab until the upload is complete. You may still navigate the console freely.`